src/west/app/project.py

```diff
@@ -1670,16 +1671,15 @@ def do_add_parser(self, parser_adder):
         parser.add_argument('projects', metavar='PROJECT', nargs='*',
                             help='''projects (by name or path) to operate on;
                             defaults to active cloned projects''')
+        parser.add_argument('-j', '--jobs', nargs='?', const=-1,
```

Comment:
```suggestion
        # Default to 1 when `-j` is not given because there is no way to
        # whether the user commands can be run at the same time safely.
        parser.add_argument('-j', '--jobs', nargs='?', const=-1,
```
Eventually, `west grep`, `west update` and others could default to `cpu_count()` if everything goes well but I think `forall` should always default to 1.

(such a comment also helps a bit with the peculiar `default+const` argparse idiom)